docs/src/language_servers.md

```diff
@@ -751,6 +752,33 @@ See [Javascript/TypeScript](#javascripttypescript).
     }
     ```
 
+5. Optional: to enable some useful commands provided by language server, add the following to `*.sublime-commands` (see more [here](commands.md#execute-server-commands)):
```

Comment:
I'm not sure if this link is relevant here... Why would someone who is adding this command need an explanation on executing server commands?

Reply:
Makes sense, then I will copy info about "... by adding an entry to your existing `*.sublime-commands` file or by creating a new one" here